naga/tests/out/hlsl/debug-printf-s.hlsl

```diff
@@ -0,0 +1,11 @@
+void main_1()
+{
+    printf("%d",42);
```

Comment:
I'm not able to get DXC to validate this. With a fresh build of DirectXShaderCompiler from git trunk, I get:

```
[INFO  xtask] Validating tests/out/hlsl/debug-printf-s.hlsl
warning: Promoting older shader model profile to 6.0 version.
tests/out/hlsl/debug-printf-s.hlsl:3:5: error: use of unsupported identifier 'printf'
    printf("%d",42);
    ^

[ERROR xtask::result] failed to validate entry point "main" with profile "cs_5_1"

Caused by:
    `dxc tests/out/hlsl/debug-printf-s.hlsl -T cs_5_1 -E main -Wno-parentheses-equality -Zi -Qembed_debug -Od -HV 2018` failed to run; exit code: Some(5)
[ERROR xtask::result] one or more shader HLSL shader tests failed for tests/out/hlsl/debug-printf-s.hlsl
```

What version of DXC do I need to get this to validate properly?

Reply:
Oh, I see the original comment where you said that only FXC supports this.

Could you please adapt the snapshot tests to skip that file on DXC? I think there's a mechanism for that already.